avro/swagger/src/main/java/com/cerner/beadledom/avro/SwaggerAvroModelConverter.java

```diff
@@ -54,57 +58,49 @@
  * subsequent whitespace character) will be removed. You can override the
  * {@link #adjustDescription(String)} method to change this.
  */
-public class SwaggerAvroModelConverter extends SwaggerSchemaConverter {
+public class SwaggerAvroModelConverter implements ModelConverter {
   private static final Logger LOGGER = LoggerFactory.getLogger(SwaggerAvroModelConverter.class);
 
   @Override
-  public Option<Model> read(Class<?> cls, Map<String, String> typeMap) {
-    Schema schema = getSchema(cls);
+  public Model resolve(Type type, ModelConverterContext context, Iterator<ModelConverter> chain) {
+    if (!(type instanceof Class<?>)) {
+      return chain.next().resolve(type, context, chain);
```

Comment:
`chain.next()` will throw NoSuchElementException exception if there are no more elements in the chain. You will need to check if there are more elements by calling `chain.hasNext`.

Reply:
There is always another element in the chain at that point as swagger-core has a default one that is always last.